federatedscope/core/auxiliaries/transform_builder.py

```diff
@@ -0,0 +1,44 @@
+from importlib import import_module
+import federatedscope.register as register
+
+
+def get_transform(config, package):
```

Comment:
Simple documentation can be added to tell users the input/output of the `transform`.